config/vufind/BrowZine.ini

```diff
@@ -4,6 +4,7 @@
 [General]
 ; BrowZine-issued access token for API
 access_token = "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
+;access_token_file = /path/to/secret
```

Comment:
It would probably be a good idea to add a comment like

```
; Use access_token_file to load the secret from another file instead of including it directly in this configuration.
```

(Here and in all the other affected configs).